.github/workflows/compliance.yml

```diff
@@ -77,8 +77,12 @@ jobs:
         git log  --pretty=oneline | head -n 10
         # Increase rename limit to allow for large PRs
         git config diff.renameLimit 10000
-        ./scripts/ci/check_compliance.py --annotate -e KconfigBasic -e SysbuildKconfigBasic -e ClangFormat \
-        -c origin/${BASE_REF}..
+        excludes="-e KconfigBasic -e SysbuildKconfigBasic -e ClangFormat"
+        # The signed-off-by check for dependabot should be skipped
+        if [ "${{ github.actor }}" == "dependabot[bot]" ]; then
+          excludes="$excludes -e Identity"
+        fi
+        echo "$excludes" | xargs ./scripts/ci/check_compliance.py --annotate -c origin/${BASE_REF}..
```

Comment:
bit confused, do you need xargs here? can't you just put `$excludes` at tthe end of the command line?

Reply:
So in `bash`: yes, in `zsh`: no. Using the `xargs` here makes it work for both now.

Reply:
ok... does it matter? this is in the workflow and it always uses bash

Reply:
well, sh I guess by default

Reply:
Sure, updated.